cedar-policy-validator/src/schema.rs

```diff
@@ -434,6 +436,11 @@ impl ValidatorSchema {
             }
         }
 
+        // Check for undeclared entity types within common types.
+        for common_type in common_types {
+            Self::check_undeclared_in_type(&common_type, entity_types, &mut undeclared_e);
+        }
```

Comment:
Do we also need to check for undeclared _common_ types within common types?

Reply:
That's already checked by the common type resolver ([link](https://github.com/cedar-policy/cedar/blob/eb740c26a96f5763e8662d02148d76b72327eee2/cedar-policy-validator/src/schema.rs#L805-L815))